### PR TITLE
Create a StartProcess util method

### PR DESCRIPTION
**Pull request by vguzmanp**

### CODE REVIEW

File: VAS.Core/Common/Utils.cs [hunk 1]
```diff
@@ -168,6 +170,37 @@ public static Stream GetEmbeddedResourceFileStream (string resourceId)
...
+			Process proc = new Process ();
+			proc.EnableRaisingEvents = false;
+
+			proc.StartInfo.WorkingDirectory = Path.GetFullPath (Path.Combine (App.Current.baseDirectory, "bin"));
+			foreach (string variable in environmentVariables.Keys) {
+				proc.StartInfo.EnvironmentVariables [variable] = environmentVariables [variable];
+			}
+
+			if (System.Environment.OSVersion.Platform == PlatformID.Win32NT) {
+				proc.StartInfo.FileName = Path.Combine (App.Current.baseDirectory, "bin", $"{filename}.{windowsExtension}");
```
Comment (ylatuya) [inline VAS.Core/Common/Utils.cs]: the first part of the path can be cached or taken from proc.StartInfo.WorkingDirectory.

Comment (ylatuya) [inline VAS.Core/Common/Utils.cs]: Extension must be ".exe" instead of "exe". Why do you need it anyway ? I wouldn't use platform specific arguments.

Comment (vguzmanp) [inline VAS.Core/Common/Utils.cs]: I was going to just add ".exe" to the filename, but I thought that someone could need to run a different thing.
Rechecking the docs, using `UseShellExecute = false`we can only run exes, so we don't need it.

Comment (vguzmanp) [inline VAS.Core/Common/Utils.cs]: fixed

**Pull request merged**